### PR TITLE
made a contact page

### LINKED ISSUE #48: As a User i want to be able to visit a contact page so i can contact the admin if i have a question.
Acceptance criteria:
Given: every page consist of a navigation bar with a contact page link.
When: the user clicks on the contact page link.
Then: the user will see the page with contact info.

### PR DESCRIPTION
**Pull request by avanheeren**
Made a contact page, added it to navbar, contact page contains info about Joeri with a picture next to it

### CODE REVIEW

File: resources/views/contact.blade.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                            <img style="height: 360px; width: 360px; margin: 10px 50px 50px 180px;" src="./image/contact.png" />
+                        </div>
+                    </div>
+                </section>
+
+
+            </div>
+        </div>
+    </section>
+
```
Comment (awqfrxx) [inline resources/views/contact.blade.php]: fix whitespace

File: resources/views/contact.blade.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <section class="hero  is-medium  is-bold">
+                    <div class="header-img columns">
+                        <div class="column">
+                            <img style="height: 360px; width: 360px; margin: 10px 50px 50px 180px;" src="./image/contact.png" />
+                        </div>
+                    </div>
+                </section>
+
+
```
Comment (awqfrxx) [inline resources/views/contact.blade.php]: fix whitespace

File: resources/views/contact.blade.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                            </a>
+
+                        </div>
+                    </div>
+                </div>
+
+                <section class="hero  is-medium  is-bold">
+                    <div class="header-img columns">
+                        <div class="column">
+                            <img style="height: 360px; width: 360px; margin: 10px 50px 50px 180px;" src="./image/contact.png" />
```
Comment (awqfrxx) [inline resources/views/contact.blade.php]: fix in-line styling

**Review by awqfrxx — CHANGES_REQUESTED** [review]
page looks good, just fix whitespace and in-line styling

File: resources/views/common/master.blade.php [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>SDG Tool</title>
 
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
```
Comment (Koreyoshi159) [inline resources/views/common/master.blade.php]: Why is this in masters.blade.php you only use it on the contact page right?

File: resources/views/contact.blade.php [hunk 5]
```diff
@@ -0,0 +1,67 @@
+@extends('common.master')
+
+@section('content')
+    <div class="columns" style="margin: 50px">
```
Comment (awqfrxx) [inline resources/views/contact.blade.php]: there's also inline styling here and below these as well

**Review by koenhamstra — APPROVED** [review]
Looks good

**Review by awqfrxx — APPROVED** [review]

**Pull request merged**